src/util/db/conflict-resolution.ts

```diff
@@ -0,0 +1,195 @@
+import { asArray, asMaybe } from 'cleaners'
+import nano from 'nano'
+
+import { AppState } from '../../server'
+import {
+  asStoreFileDocument,
+  StoreDocument,
+  StoreFileDocument
+} from '../../types/db-types'
+
+// Response Types for CouchDB's _bulk_get API:
+
+interface BulkGetResponse {
+  results: BulkGetResult[]
+}
+interface BulkGetResult {
+  id: string
+  docs: Array<BulkGetResultDoc | BulkGetResultError>
+}
+interface BulkGetResultDoc {
+  ok: BulkGetDoc
+}
+type BulkGetDoc = StoreDocument
+
+interface BulkGetResultError {
+  error: BulkGetError
+}
+interface BulkGetError {
+  id: string
+  rev: string
+  error: string
+  reason: string
+}
+
+export const resolveAllDocumentConflicts = (appState: AppState) => async (
+  repoId: string
+): Promise<void> => {
+  // Query for document IDs with conflicts using the conflictRevs view
+  const conflictQueryResponse = await appState.dataStore.partitionedView(
+    repoId,
+    'conflicts',
+    'conflictRevs',
+    {
+      include_docs: false
+    }
+  )
+  const docIds = conflictQueryResponse.rows.map(row => ({ id: row.id }))
+
+  // Query for the conflicting documents for each document ID using _bulk_get
+  const documentsResponse: BulkGetResponse = await appState.dbServer.request({
```

Comment:
I think we should make a `bulkGet(db: nano.DocumentScope, docs: string[])`, since it's not part of nano, and put it in its own unit file along with types. Maybe it could even go into edge-server-utils with the other couch tools.